components/eamxx/src/share/field/field_impl.hpp

```diff
@@ -197,6 +197,15 @@ deep_copy_impl (const Field& src) {
        "ERROR: Unable to copy field " + src.get_header().get_identifier().name() +
           " to field " + get_header().get_identifier().name() + ".  Layouts don't match.");
   const auto  rank = layout.rank();
+
+  // For rank 0 view, we only need to copy a single value and return
+  if (rank == 0) {
+    auto v     =     get_view<      ST,HD>();
+    auto v_src = src.get_view<const ST,HD>();
+    v() = v_src();
+    return;
+  }
```

Comment:
@bartgol No rush, just when you have a chance, let me know what you think about me exiting early here to avoid computing extents and policies etc. Rank 0 can just copy it's value and return.

Reply:
Yeah, early return is always ok (even though you should never trust people saying "always" in programming...).